internal/services/controller/controller_exclude_race_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build !race
```

Comment:
per Valdas' comment in discord, this was put into no-race run mode due to flakyness